### PR TITLE
[MO] - [system test] -> creating topics via adminclient

### PR DESCRIPTION
**Pull request by see-quick**
Signed-off-by: morsak <xorsak02@stud.fit.vutbr.cz>

### Type of change

- Enhancement / new feature

### Description

This PR adding new scalabillity test for the topics with using of AdminClient API.

### Checklist

- [x] Write tests
- [x] Make sure all tests pass

### CODE REVIEW

File: systemtest/src/test/java/io/strimzi/systemtest/TopicST.java [hunk 1]
```diff
@@ -181,6 +195,45 @@ void testBigAmountOfTopicsCreatingViaKafka() {
...
+                        .withNewKafkaListenerExternalNodePort()
+                            .withTls(false)
+                        .endKafkaListenerExternalNodePort()
+                    .endListeners()
+                .endKafka()
+            .endSpec()
+            .done();
+
+        Properties properties = new Properties();
+        properties.setProperty(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, AbstractKafkaClient.getExternalBootstrapConnect(NAMESPACE, CLUSTER_NAME));
```
Comment (ppatierno) [inline systemtest/src/test/java/io/strimzi/systemtest/TopicST.java:216]: why the admin client goes through an external listener?

File: systemtest/src/test/java/io/strimzi/systemtest/TopicST.java [hunk 2]
```diff
@@ -181,6 +195,45 @@ void testBigAmountOfTopicsCreatingViaKafka() {
...
+                crt.all().get();
+            }
+
+            Set<String> topics = adminClient.listTopics().names().get(Constants.GLOBAL_CLIENTS_TIMEOUT, TimeUnit.MILLISECONDS);
+
+            LOGGER.info("Verify that in Kafka cluster contains {} topics", NUMBER_OF_TOPICS);
+            assertThat(topics.size(), is(NUMBER_OF_TOPICS));
+        }
+
+        LOGGER.info("Verify that {} topics are inside Kubernetes CR", NUMBER_OF_TOPICS);
```
Comment (ppatierno) [inline systemtest/src/test/java/io/strimzi/systemtest/TopicST.java:233]: I would say "Verify that corresponding {} KafkaTopic custom resources were created"

File: systemtest/src/test/java/io/strimzi/systemtest/TopicST.java (hunk 1 above)
Comment (scholzj) [reply systemtest/src/test/java/io/strimzi/systemtest/TopicST.java:216]: The client runs locally, or?

Comment (see-quick) [reply systemtest/src/test/java/io/strimzi/systemtest/TopicST.java:216]: The client runs locally.

File: systemtest/src/test/java/io/strimzi/systemtest/TopicST.java [hunk 3]
```diff
@@ -181,6 +195,45 @@ void testBigAmountOfTopicsCreatingViaKafka() {
         }
     }
 
+    @Tag(NODEPORT_SUPPORTED)
+    @Tag(SCALABILITY)
+    @Test
+    void testBigAmountOfTopicsCreatingViaAdminClient() throws ExecutionException, InterruptedException, TimeoutException {
+
+        KafkaResource.kafkaEphemeral(CLUSTER_NAME, 3, 1)
```
[the remark this answers is not in the document]
Comment (see-quick) [reply systemtest/src/test/java/io/strimzi/systemtest/TopicST.java:203]: Yes. Do i need more?

File: systemtest/src/test/java/io/strimzi/systemtest/topic/TopicST.java [hunk 4]
```diff
@@ -106,6 +118,41 @@ void testCreateTopicViaKafka() {
...
+        try (AdminClient adminClient = AdminClient.create(properties)) {
+
+            LOGGER.info("Creating async topic {} via Admin client", TOPIC_NAME);
+            CreateTopicsResult crt = adminClient.createTopics(singletonList(new NewTopic(TOPIC_NAME, 1, (short) 1)));
+            crt.all().get();
+
+            Set<String> topics = adminClient.listTopics().names().get(Constants.GLOBAL_CLIENTS_TIMEOUT, TimeUnit.MILLISECONDS);
+
+            LOGGER.info("Verify that in Kafka cluster contains {} topics", 1);
+            assertThat(topics.size(), is(1));
```
Comment (ppatierno) [inline systemtest/src/test/java/io/strimzi/systemtest/topic/TopicST.java:149]: asserThat it contains exactly `TOPIC_NAME` other than just the size?

File: systemtest/src/test/java/io/strimzi/systemtest/topic/TopicST.java [hunk 5]
```diff
@@ -106,6 +118,41 @@ void testCreateTopicViaKafka() {
...
+        }
+
+        LOGGER.info("Verify that corresponding {} KafkaTopic custom resources were created", 1);
+        assertThat(KafkaTopicResource.kafkaTopicClient().inNamespace(NAMESPACE).list().getItems().size(), is(1));
```
Comment (ppatierno) [inline systemtest/src/test/java/io/strimzi/systemtest/topic/TopicST.java:153]: the same, assertThat the metadata.name is `TOPIC_NAME`?

**Pull request merged**